src/Service.Tests/SqlTests/GraphQLQueryTests/MsSqlGraphQLQueryTests.cs

```diff
@@ -374,6 +375,22 @@ public async Task QueryAgainstSPWithOnlyTypenameInSelectionSet()
             await QueryAgainstSPWithOnlyTypenameInSelectionSet(dbQuery);
         }
 
+        /// <summary>
+        /// Checks failure on providing arguments with no default in runtimeconfig.
+        /// </summary>
```

Comment:
Can you elaborate on the state of the system under test in the comment:
1. Which fields are expected to be present in the request for the request to pass? 
2. What runtime config properties are included/excluded to ensure this test is in the proper state? e.g. no default values specified in runtime config 
3. What are the relevant db schema traits of the stored procedure under test? e.g. defines X number of default value params which are named a, b, and c.

I think one more test is warranted where the GraphQL request includes the a value for the field, where the field:
- has default value defined in db
- no default value defined in config

Result: successful response.

Reply:
This test seems to be already confirmed by this test case right: 
![image](https://github.com/Azure/data-api-builder/assets/124841904/73bff85b-6441-4095-ad22-39a2ad170651)

Reply:
The test in the screenshot may have unintentionally covered the use case I mentioned. 

If that pre-existing test has its underlying entity swapped out, then covering this test case may no longer apply, even though the test in the screenshot still accomplishes what it claims to be testing :

> The GraphQL request includes a value for the field, where the field:
> 
> - has default value defined in db
> - no default value defined in config

-> Action: Update the comment on that test `TestStoredProcedureQueryForGettingSingleRow(string dbQuery)` to elaborate on the state of the system under test like in my first comment of this thread.

Reply:
Makes sense. added comment.